client/dashboard/data/site-themes.ts

```diff
@@ -0,0 +1,12 @@
+import wpcom from 'calypso/lib/wp';
+
+export interface Theme {
+	is_block_theme: boolean;
+}
+
+export async function fetchSiteThemesActive( siteId: number ) {
```

Comment:
Nit: I think we should prefer (more) grammatically correct English instead of trying to make the prefix consistent, so can we use `fetchSiteActiveTheme()`, and `siteActiveThemeQuery()` instead? Similar to existing `siteCurrentPlanQuery()` etc

Reply:
I'm conflicted on this, as the file name is site-themes so it also feels right that it follows the naming. Also, the query returns an array implying multiple themes. How about something like fetchSiteThemesOnlyActive?

Reply:
>  as the file name is site-themes so it also feels right that it follows the naming.

But when using the functions (es, devs will only read the name without association to the filename 😄  It's similar case to why we usually call `UnforwardedButton` vs `ButonUnforwarded` as the former is the grammatically correct one.

> Also, the query returns an array implying multiple themes

Hmmm.... right...

- `fetchSiteActiveThemes()`
- `siteActiveThemesQuery()` 

then 😄

Reply:
No strong preference on my end, so I'm happy to update it 🙂 813bec1480165777b9ed80349f1b865fde741d49.